src/modules/LanUdevAuto.rb

```diff
@@ -184,7 +184,7 @@ def Import(settings)
           createUdevFromIfaceName(Ops.get_list(settings, "interfaces", []))
         )
       else
-        @udev_rules = Ops.get_list(settings, "net-udev", [])
+        @udev_rules = settings["net-udev"] || {}
```

Comment:
shouldn't this be ``settings["net-udev"] || []`. Looking at the rest of the code, it seems to be an array of hashes, not a hash.